fem/coefficient.hpp

```diff
@@ -403,15 +470,24 @@ class VectorGridFunctionCoefficient : public VectorCoefficient
    GridFunction *GridFunc;
 
 public:
+   /** @brief Construct an empty coefficient.  Calling Eval() before the grid
+       function is set with cause a segfault. */
    VectorGridFunctionCoefficient() : VectorCoefficient(0), GridFunc(NULL) { }
+
+   /** @brief  Construct the coefficient with grid function @a gf.  The
+       grid function is not owned by the coefficient. */
    VectorGridFunctionCoefficient(GridFunction *gf);
 
+   /// Set the grid function
    void SetGridFunction(GridFunction *gf);
    GridFunction * GetGridFunction() const { return GridFunc; }
```

Comment:
Missing description

Reply:
Do we really need comments for every single public method?

Reply:
For example what's the point of this comment?
```
/// Set the grid function
void SetGridFunction(GridFunction *gf);
```